app/routes/admin/groups/components/GroupMembers.js

```diff
@@ -89,13 +89,13 @@ export const GroupMembers = ({
 );
 
 function loadData({ query, match: { params }, location }, dispatch) {
-  const showDescendants = location.search.includes('descendants=true');
+  const showDescendants = window.location.search.includes('descendants=true');
```

Comment:
Hmmm, isn't this using the location param from above? Using `window` will break ssr. Or am I missing something?

Reply:
Hm, eslint does not complain about the use of only location in this case. Maybe it is okay to just leave it to ```const showDescendants = location.search.includes('descendants=true');```javascript here, and just refer to search below?